code/modules/mob/inventory.dm

```diff
@@ -77,25 +71,12 @@
 	W.plane = initial(W.plane)
 	W.dropped()
 
-/mob/proc/drop_item_v()		//this is dumb.
-	if(stat == CONSCIOUS && isturf(loc))
-		return drop_item()
-	return 0
-
-//Drops the item in our left hand
-/mob/proc/drop_l_hand()
-	return unEquip(l_hand) //All needed checks are in unEquip

```

Comment:
If you really wanted to keep it, I could alter it to do
`drop_l_hand(var/force)
     return unEquip(l_hand force = 0)`
But even that would require the same sort of changes because the two functions are used interchangeably in the code.